.gitignore

```diff
@@ -8,6 +8,8 @@ logs
 pids
 *.pid
 *.seed
+.sass-cache/
+hunt creation
```

Comment:
What is this line? I do not think you can have a space on a line like this for a git ignore.